docs/examples/zh/loaders/GLTFLoader.html

```diff
@@ -134,7 +134,7 @@ <h2>纹理</h2>
 		颜色空间并显示在屏幕上。除非你需要使用线性颜色空间进行后期处理，否则请在使用glTF的时候将[page:WebGLRenderer]进行如下配置：</p>
 
 		<code>
-		renderer.outputEncoding = THREE.sRGBEncoding;
+		renderer.outputColorSpace = THREE.sRGBEncoding;
```

Comment:
It's OK with me to keep this for another PR but I do think it's important that we make sure `.outputColorSpace` accepts the color space enum values, within the same release as this PR.

Optionally — we could change the _values_ of the encoding enums, such that `THREE.sRGBEncoding` and `THREE.LinearEncoding` are interchangeable with their color space equivalents, during the transition.